### PR TITLE
WebHost: games played per day plot per game on stats page

### PR DESCRIPTION
**Pull request by Berserker66**
built on top of changes from https://github.com/ArchipelagoMW/Archipelago/pull/825

### CODE REVIEW

File: WebHostLib/stats.py [hunk 1]
```diff
@@ -65,12 +112,15 @@ def stats():
...
-    pie.wedge(x=0.5, y=0.5, radius=0.5,
+    data["colors"] = [game_to_color[game] for game in data["games"]]
+
+    pie.wedge(x=0, y=0, radius=0.5,
               start_angle="start_angles", end_angle="end_angles", fill_color="colors",
               source=ColumnDataSource(data=data), legend_field="games")
 
-    script, charts = components((plot, pie))
+    per_game_charts = [create_game_played_figure(games_played, game, game_to_color[game]) for game in total_games
+                       if total_games[game] > 1]
```
Comment (black-sliver) [inline WebHostLib/stats.py:122]: any particular reason for the `>1` ? This made my testing awkward

File: WebHostLib/stats.py [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
 from bokeh.plotting import figure, ColumnDataSource
 from bokeh.resources import INLINE
+from bokeh.colors import RGB
 from flask import render_template
 from pony.orm import select
 
 from . import app, cache
 from .models import Room
 
+PLOT_WIDTH = 600
```
Comment (black-sliver) [inline WebHostLib/stats.py:18]: we may want to look into more responsive design, but this is probably more of a web design question.

**Review by black-sliver — APPROVED** [review]
Left 2 comments below.

Looks good to me, but you may want to get a second opinion from someone that enjoys web dev more than I do.

File: WebHostLib/stats.py (hunk 1 above)
Comment (Berserker66) [reply WebHostLib/stats.py:122]: A graph with a single datapoint just becomes a big rectangle in bokeh vbars

File: WebHostLib/stats.py (hunk 2 above)
Comment (Berserker66) [reply WebHostLib/stats.py:18]: yeah, outside my area too, but in my opinion they look better like that... on my screen.
They support responsive but then the site also has to support that. Hopefully a webdesigner takes a shot at that at some point.

**Pull request merged**